aerospike/manifest.json

```diff
@@ -4,7 +4,8 @@
   "max_agent_version": "6.0.0",
```

Comment:
I believe you'd be happy to be the maintainer for the check 🙇  if that's still the case, could you update the `maintainer` field in the manifest with your email contact info? Thank you!